Database/osirose.sql

```diff
@@ -236,7 +236,8 @@ CREATE TABLE `skill` (
   `char_id` int(10) unsigned NOT NULL DEFAULT '0',
   `id` smallint(5) unsigned NOT NULL DEFAULT '0',
   `level` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  PRIMARY KEY (`char_id`,`id`)
+  PRIMARY KEY (`char_id`,`id`),
+  CONSTRAINT `char_id` FOREIGN KEY (`char_id`) REFERENCES `characters` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```

Comment:
Doesn't it have to be a constraint? Isn't it enough to declare it as foreign key?

Reply:
Apparently this is valid syntax but redundant. I will accept and up to you if you want to change it or not

Reply:
It's redundant but used in the whole file, so I'd go for consistency here.

Reply:
Actually it poses problems so I'm removing it everywhere.

Reply:
the reason why it's like that is because that is what the mysql workbench tool made it.